data-reconciliation-app/test/test.sh

```diff
@@ -88,29 +88,70 @@ cd "${certificate_dir}"
 # -------------------------- Test cases --------------------------
 echo "Test start"
 
+printf "\n  -------- Test Ingestion Service --------  \n\n"
+
 ingestUrl="$server/app/ingest";
 
 memberName="member0"
-check_eq "$memberName - data ingest succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "@../../data-samples/${memberName}_data.json" $only_status_code)"
+check_eq "$memberName - data ingest should succeed" "200" "$(curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "@../../test/data-samples/${memberName}_data.json" $only_status_code)"
+printf " Response: "
+curl $ingestUrl -X POST $(cert_arg $memberName) -H "Content-Type: application/json" --data-binary "@../../test/data-samples/${memberName}_data.json"
+printf "\n\n"
```

Comment:
Can you refresh me the reason for having the response of the ingest API printed on the terminal? is it relevant for our tests?  
It's just a json object with body message as `data ingested successfully` right?